pkg/dbutils/dbutils.go

```diff
@@ -80,3 +86,83 @@ func QuoteLiteral(str string) string {
 		return `'` + res + `'`
 	}
 }
+
+func InitPublication(conn *pgx.Conn, publicationName string) error {
```

Comment:
I'd name it CreatePublicationIfNotExists, as we don't do any actions if the publication is there.